src/test/java/org/pac4j/dropwizard/factory/FakeConfig.java

```diff
@@ -0,0 +1,24 @@
+package org.pac4j.dropwizard.factory;
+
+import org.pac4j.jax.rs.pac4j.JaxRsConfig;
+
+import java.util.Map;
+
+/**
+ * Fake configuration for tests.
+ *
+ * @author Jerome Leleu
+ * @since 2.0.0
+ */
+public final class FakeConfig extends JaxRsConfig {
+
+    private Map<String, String> properties;
+
+    public Map<String, String> getProperties() {
```

Comment:
ah, ok, so it is to verify that FakeConfigFactory has the parameters passed to it as expected! Ok I got it, thanks :)